### ISSUE TITLE
[Glossary] Region

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1970]
https://www.w3.org/TR/WCAG21/#dfn-regions

> In HTML, any area designated with a landmark role would be a region.

should be:

> In HTML, any area designated with a landmark role is a region.

**Comment by alastc** [comment]
Hi @yatil, in the context of a note the wording makes sense to me because it is saying "This is the definition, and this is an example that would match the definition".

Considering there is no change of meaning, it is worth updating?

**Comment by yatil** [comment]
I had the question from young auditors if all landmark roles are a region or if there are differences, based on this note. Why be ambiguous when we cannot?

**Comment by JAWS-test** [comment]
There are two problems with the definition:
1. there are no landmark roles in HTML, only in ARIA (the term does not appear anywhere in the entire HTML specification).
2. HTML elements that are assumed to represent landmark regions (like header and footer) rarely result in a region (see e.g. https://www.w3.org/TR/html-aam-1.0/#details-id-47)
